pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -404,33 +388,32 @@ func isNodeAboveTargetUtilization(usage NodeUsage) bool {
 // isNodeWithLowUtilization checks if a node is underutilized
 // All resources have to be below the low threshold
 func isNodeWithLowUtilization(usage NodeUsage) bool {
-	for name, nodeValue := range usage.usage {
-		// usage.lowResourceThreshold[name] < nodeValue
-		if usage.lowResourceThreshold[name].Cmp(*nodeValue) == -1 {
-			return false
+	for name, threshold := range usage.lowResourceThreshold {
+		if nodeValue, ok := usage.usage[name]; ok {
+			// usage.lowResourceThreshold[name] < nodeValue
+			if threshold.Cmp(*nodeValue) == -1 {
+				return false
+			}
 		}
 	}
 
 	return true
 }
 
+// Sum up resource requests (cpu, memory) and the number of pods
+// All resource are in MilliValue (value * 1000)
 func nodeUtilization(node *v1.Node, pods []*v1.Pod) map[v1.ResourceName]*resource.Quantity {
-	totalReqs := map[v1.ResourceName]*resource.Quantity{
```

Comment:
These lines can stay as well for the reasons mention earlier.

Reply:
@ingvagabund, commits updated, would you please help review again.